messaging/src/main/java/org/axonframework/eventhandling/EventTrackerStatus.java

```diff
@@ -49,6 +51,21 @@
      */
     boolean isReplaying();
 
+    /**
+     * Indicates whether this Segment is still merging events.
+     *
+     * @return {@code true} if this segment is merging events, otherwise {@code false}
+     */
+    boolean isMerging();
+
+    /**
+     * Return the estimated relative token position this Segment will have after a merge operation is complete.
+     * In case no estimation can be given or no merge in progress, an {@code OptionalLong.empty()} will be returned.
+     *
```

Comment:
Suggestion: might be nice to mention on this piece of javadoc that this method will return a non-empty result as long as `isMerging` returns true.

Reply:
Think this line could still be added? :angel:

Reply:
Oh sorry I forgot to reply this one :man_facepalming: 
I feel that is not completely true : a segment can return true for isMerging(), but will return OptionalLong.empty() in case no estimation can be given.